File: resource_management/resource_management.service/resource_management.service.discovery/src/main/java/org/eclipse/slm/resource_management/service/discovery/api/DiscoveryJobStartedResponse.java
package org.eclipse.slm.resource_management.service.discovery.api;

import java.util.UUID;

public class DiscoveryJobStartedResponse {

    private UUID discoveryJobId;

    public DiscoveryJobStartedResponse() {
    }

    public DiscoveryJobStartedResponse(UUID discoveryJobId) {
        this.discoveryJobId = discoveryJobId;
    }

    public UUID getDiscoveryJobId() {
        return discoveryJobId;
    }

    public void setDiscoveryJobId(UUID discoveryJobId) {
        this.discoveryJobId = discoveryJobId;
    }
}

File: resource_management/resource_management.features/resource_management.features.device-integration/resource_management.features.device-integration.discovery/src/main/java/org/eclipse/slm/resource_management/features/device_integration/discovery/api/DiscoveryJobStartedResponse.kt
package org.eclipse.slm.resource_management.features.device_integration.discovery.api

import java.util.*

class DiscoveryJobStartedResponse {

    var discoveryJobId: UUID? = null

    constructor()

    constructor(discoveryJobId: UUID?) {
        this.discoveryJobId = discoveryJobId
    }

}
